ios/Podfile

```
@@ -85,102 +69,95 @@
-        cfg.build_settings['ENABLE_USER_SCRIPT_SANDBOXING'] = 'NO'
+        bs = cfg.build_settings
+
+        # === CRITICAL FIX FOR HEADER RESOLUTION ===
+        # Make header maps and searches robust so <react/bridging/*.h> resolves.
+        bs['USE_HEADERMAP'] = 'YES'
+        header_searches = [
+          '$(inherited)',
+          '$(PODS_ROOT)/Headers/Public/**',
+          '$(PODS_ROOT)/Headers/Private/**',
+          # Some RN pods generate headers into the config build dir as well:
+          '$(PODS_CONFIGURATION_BUILD_DIR)/**'
```

RuboCop (1.76.1)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)